### PR TITLE
Mention wiki in README

### PR DESCRIPTION
**Pull request by MaxLeiter**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -54,7 +54,7 @@ When the install is complete, go ahead and run this in your terminal:
 lounge --help
 ```
 
-For more information, read the [documentation](https://thelounge.github.io/docs/).
+For more information, read the [documentation](https://thelounge.github.io/docs/) and/or [wiki](https://github.com/thelounge/lounge/wiki)
```
Comment (astorije) [inline README.md]: Could you re-add the dot? Also while you're at it, `or` instead of `and/or`.

**Comment by astorije** [comment]
Ping?

**Comment by astorije** [comment]
@MaxLeiter, squash your commits and I'm a 👍.

**Comment by MaxLeiter** [comment]
Poke @astorije

**Comment by astorije** [comment]
Thanks and sorry!

**Pull request merged**